### ISSUE TITLE
pdsh and localhost

### ISSUE THREAD

**Issue by paolobenve** [chaos/pdsh#149]
I'm on xubuntu 22.04
I'm trying to do something like

`pdsh -w localhost,otherhost command`

But the command isn't executed on localhost.

Does it have to do with the fact that `ssh localhost command` doesn't work?

**Comment by grondo** [comment]
Yes, `pdsh` is doing essentially `ssh localhost command`, so if that doesn't work then `pdsh -w localhost command` won't either.
